EED_Multi_Event_Registration.module.php

```diff
@@ -1208,9 +1210,53 @@ protected function can_purchase_ticket_quantity( $ticket = null, $quantity = 1,
 			EE_Error::add_error( $limit_error_1 . '<br/>' . $limit_error_2, __FILE__, __FUNCTION__, __LINE__ );
 			return false;
 		}
+
+		// is the quantity allowed based on ticket max?
+		if( $line_item instanceof EE_Line_Item && $ticket instanceof EE_Ticket )
+        {
+            if($action == 'add') $new_quantity = ($line_item->quantity() + $quantity);
+            else if($action == 'remove') $new_quantity = ($line_item->quantity() - $quantity);
+            else $new_quantity = $quantity;
```

Comment:
please also use explicit bracketing for formatting:

```suggestion
            $new_quantity = $quantity;
            if ($action === 'add') {
                $new_quantity = ($line_item->quantity() + $quantity);
            } elseif ($action === 'remove') {
                $new_quantity = ($line_item->quantity() - $quantity);
            }
```